website/content/api-docs/system/internal-counters.mdx

```diff
@@ -247,21 +231,19 @@ returned by the API for backward compatibility and it may be removed in the futu
 `non_entity_clients` field instead. The `non_entity_tokens` field is currently
 returned by the API for backward compatibility, and it may be removed in the future.
 
```

Comment:
This was the only conflict in the cherry-picks - the fields aren't deprecated on 1.17, so I kept those deprecation warnings